dev/cnf/dependabot/check_this_in_if_it_changes/pom.xml

```diff
@@ -351,6 +351,11 @@
       <artifactId>FastInfoset</artifactId>
       <version>1.2.16</version>
     </dependency>
+    <dependency>
```

Comment:
Does this entry need to be here if we've updated the old one to the new version?